dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       },
       {
         "value": "518GB",
-        "label": "HAVERHILL VA CLINIC"
+        "label": "Haverhill VA Clinic"
       },
       {
         "value": "518GE",
-        "label": "GLOUCESTER VA CLINIC"
+        "label": "Gloucester VA Clinic"
       },
       {
         "value": "518GG",
         "label": "FITCHBURG CBOC"
       },
       {
         "value": "523A4",
-        "label": "VA BOSTON HEALTHCARE SYSTEM - WEST ROXBURY DIVISION"
+        "label": "West Roxbury VA Medical Center"
       },
       {
         "value": "523A5",
-        "label": "VA BOSTON HEALTHCARE SYSTEM - BROCKTON DIVISION"
+        "label": "Brockton VA Medical Center"
       },
       {
         "value": "523BY",
-        "label": "VA BOSTON HEALTHCARE SYSTEM - LOWELL DIVISION"
+        "label": "Lowell VA Clinic"
       },
       {
         "value": "523BZ",
-        "label": "VA BOSTON HEALTHCARE SYSTEM - BOSTON OPC DIVISION"
+        "label": "Causeway VA Clinic"
       },
       {
         "value": "523GA",
-        "label": "VA BOSTON HEALTHCARE SYSTEM - FRAMINGHAM CBOC"
+        "label": "Framingham VA Clinic"
       },
       {
         "value": "523GB",
         "label": "VA BOSTON HEALTHCARE SYSTEM - WORCESTER DIVISION"
       },
       {
         "value": "523GC",
-        "label": "VA BOSTON HEALTHCARE SYSTEM - QUINCY CBOC"
+        "label": "Quincy VA Clinic"
       },
       {
         "value": "523GD",
-        "label": "PLYMOUTH CBOC"
+        "label": "Plymouth VA Clinic"
       },
       {
         "value": "631BY",
-        "label": "SPRINGFIELD OPC"
+        "label": "Springfield VA Clinic"
       },
       {
         "value": "631GC",
-        "label": "PITTSFIELD CBOC"
+        "label": "Pittsfield VA Clinic"
       },
       {
         "value": "631GD",
-        "label": "GREENFIELD CBOC"
+        "label": "Greenfield VA Clinic"
       },
       {
         "value": "650GA",
-        "label": "NEW BEDFORD CBOC"
+        "label": "New Bedford VA Clinic"
       },
       {
         "value": "650GB",
-        "label": "HYANNIS CBOC"
+        "label": "Hyannis VA Clinic"
       },
       {
         "value": "631GE",
-        "label": "WORCESTER CBOC"
+        "label": "Worcester VA Clinic"
       },
       {
         "value": "631GF",
-        "label": "FITCHBURG CBOC"
+        "label": "Fitchburg VA Clinic"
       },
       {
         "value": "631QB",
-        "label": "LAKE AVENUE VA CLINIC"
+        "label": "Lake Avenue VA Clinic"
       },
       {
         "value": "518",
-        "label": "EDITH NOURSE ROGERS VAMC"
+        "label": "Edith Nourse Rogers Memorial Veterans' Hospital"
       },
       {
         "value": "523",
-        "label": "VA BOSTON HEALTHCARE SYSTEM - BOSTON DIVISION"
+        "label": "Jamaica Plain VA Medical Center"
       },
       {
         "value": "631",
-        "label": "VA CNTRL WSTRN MASSCHUSETS HCS"
+        "label": "Edward P. Boland Department of Veterans Affairs Medical Center"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.

Reply:
Flagging this label since it goes from an entire health care system to a single facility.